README.md

```diff
@@ -23,6 +23,14 @@ It was created as an answer to deeply recursive Pyanodon recipes, which the tool
 
 Among other things YAFC has Never Enough Items, which is FNEI on steroids. In addition to showing the recipes, it shows which ones you probably want to use, and how much.
 
+## Getting Started
```

Comment:
I think if we want to expand on how to get started with YAFC, we want to work on the [Download](https://github.com/have-fun-was-taken/yafc-ce/blob/0aace81ce5b9f1d3d91054b4598db2845a64803c/README.md?plain=1#L58) part of Readme.

Essentially, I agree that the Download section can be moved up and changed into the steps you outlined here.

Reply:
I tried to take your suggestion into account and I've pushed up.  Let me know what you think!